blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/CalendarClock.java

```diff
@@ -0,0 +1,21 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.joda.time.Instant;
+
+import java.util.Calendar;
+
+public class CalendarClock implements Clock {
```

Comment:
`DefaultClockImpl` returns `Instant.now()`, which returns an `Instant` initialized from other sources within jodatime. I think that `Instant` by default ultimately gets its values from `System.currentTimeMillis()`, but I'm not entirely sure.

The code that I replaced, however, used `Calendar.getInstance().getTimeInMillis()`. I dug around inside `Calendar` for a while, but I couldn't find where it was getting its values from by default.

I can't be sure that `DefaultClockImpl` would provide exactly the same behavior in all cases, especially considering that the two time sources can be configured separately. For example, I'm not sure if the same time zone is used in both cases. Thus, in order to ensure that the refactoring is the same, I decided to create another clock implementation. That way, if the two time libraries initialize differently, or if some other distant part of blueflood changes time configurations somewhere somehow, it will still behave exactly as before.